src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -0,0 +1,303 @@
+import { Component } from 'react';
+import { hh, p, div, h2, span, a, button } from 'react-hyperscript-helpers';
+import { Panel } from '../components/Panel';
+import { InputFieldText } from '../components/InputFieldText';
+import { MultiSelect } from '../components/MultiSelect';
+import { Fundings } from '../components/Fundings';
+
+import { InputYesNo } from '../components/InputYesNo';
+import { InputFieldTextArea } from '../components/InputFieldTextArea';
+import { InputFieldRadio } from '../components/InputFieldRadio';
+import { Project } from "../util/ajax";
+
+const ORSP_ROLE = "orsp";
+
+class ProjectReview extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+        description: '',
+        projectExtraProps: {
+        projectTitle: '',
+        protocol: '',
+        subjectProtection: '',
+        projectReviewApproved: false
+      },
+      piList: [{ key: '', label: '', value: '' }],
+      pmList: [{ key: '', label: '', value: '' }],
+      fundings: [{ source: { label: '', value: '' }, sponsor: '', identifier: '' }],
+      collaborators: [{ key: '', label: '', value: '' }],
+      requestor: {
+        displayName: '',
+        emailAddress: ''
+      }
+    }
+  }
+
+  componentDidMount() {
+    Project.getProject(this.props.projectUrl, this.props.projectKey).then(
+      element =>
+        this.setState(prev => {
+          prev.description = element.data.issue.description;
+          prev.projectExtraProps = element.data.extraProperties;
+          prev.piList = this.getUsersArray(element.data.pis);
+          prev.pmList = this.getUsersArray(element.data.pms);
+          prev.collaborators = this.getUsersArray(element.data.collaborators);
+          prev.fundings = this.getFundingsArray(element.data.fundings);
+          prev.requestor = element.data.requestor !== null ? element.data.requestor : this.state.requestor;
+          return prev;
+        }, () => {})
+      );
+  }
+
+  getUsersArray (array) {
+    let usersArray = [];
+    if (array !== undefined  && array !== null && array.length > 0) {
+      array.map(element => {
+        usersArray.push({
+          key: element.userName,
+          label: element.displayName + " (" + element.emailAddress + ") ",
+          value: element.displayName
+        });
+      });
+    }
+    return usersArray
+  }
+
+  getFundingsArray (fundings) {
+    let fundingsArray = [];
+    if (fundings !== undefined && fundings !== null && fundings.length > 0) {
+      fundings.map(funding => {
+        fundingsArray.push({
+          source:{
+            label: funding.source,
+            value: funding.source.split(" ").join("_").toLowerCase()
+          },
+          sponsor: funding.name,
+          identifier: funding.awardNumber !== null ? funding.awardNumber : ''
+        });
+      });
+    }
+    return fundingsArray;
+  }
+
+  isEmpty(value) {
+    return value === '' || value === null || value === undefined;
+  }
+
+  isAdmin = (e) => {
```

Comment:
In next PR, we should use UserInfo.isAdmin() method. Please research how to use it from React.

Reply:
Will do